contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -27,18 +32,18 @@ Apart from installing VueJs, you can install [VueJs development tools](https://g
 
```

Comment:
To be consistent with other similar articles, remove the section about installation.